_pages/known-issues/verify-receive-address.md

```diff
@@ -12,8 +12,8 @@ Let's say you have a `2-of-3` multisig (with seeds A, B, and C), and your host c
 Unfortunately for you, one of your hardware wallets is compromised/fake.
 Here is how you could be tricked into using an address you don't actually control to receive funds:
 
-1. Specter on your host computer (malware) displays a `2-of-3` receiving address, where *none* of your hardware wallets are actually part of the multisig scheme.
-1. You pull out hardware wallet A (also malware!) and it (falsely) confirms the same receive address on its trusted display.
+1. Specter-Desktop on your host computer (assumed to be malware infected) displays a `2-of-3` receiving address, where *none* of your hardware wallets are actually part of the multisig scheme.
+1. You pull out hardware wallet A (also malware infected!) and it (falsely) confirms the same receive address on its trusted display.
```

Comment:
This is not your fault, but what I wrote here is confusing. I'm going to rework this.